README.md

```diff
@@ -16,8 +16,6 @@ It was built with immutability and declarative configuration as code in mind.
 
 Out of the box it provides:
 - integration with Kubernetes ([Jenkins kubernetes-plugin](https://github.com/jenkinsci/kubernetes-plugin))
-- pipelines as code ([Jenkins pipelines](https://jenkins.io/doc/book/pipeline/))
```

Comment:
This is very important information for users new to Jenkins. It should stays as it is.

Reply:
I think this can be misleading based on what the operator provides and what is provided by Jenkins. I will update the PR appropriately.

Reply:
I have rewriteen this info considering that it is a preliminary to understand before consuming the operator.

Reply:
I just reword it, something like:
- `preconfigured pipelines as code Jenkins plugin...`

Reply:
bump